x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_actions/action_to_rules_client_operation.ts

```diff
@@ -87,6 +87,20 @@ export const bulkEditActionToRulesClientOperation = (
         getNotifyWhenOperation(action.value.throttle),
       ];
 
+    // schedule actions
+    case BulkActionEditType.set_schedule:
+      return [
+        {
+          field: 'schedule',
+          operation: 'set',
+          // We need to pass a pure Interval object
+          // i.e. get rid of the meta property
+          value: {
+            interval: action.value.interval,
+          },
```

Comment:
Could you expand here? Not sure I fully understand what you mean.

Reply:
This comment is actually outdated after a refactor, and also what I meant is covered by the already existing test:
```ts
  test('should transform schedule bulk edit correctly', () => {
    expect(
      bulkEditActionToRulesClientOperation({
        type: BulkActionEditType.set_schedule,
        value: {
          interval: '100m',
          lookback: '10m',
        },
      })
    ).toEqual([
      {
        field: 'schedule',
        operation: 'set',
        value: { interval: '100m' },
      },
    ]);
  });
```
basically, that the `lookback` property should be ignored.

Reply:
@jpdjere if the comment is outdated are you going to remove it in https://github.com/elastic/kibana/pull/141604?

Reply:
Yes, will add there